guides/common/modules/proc_using-red-hat-insights.adoc

```diff
@@ -12,6 +12,18 @@ For more information, see {ManagingHostsDocURL}registering-a-host_managing-hosts
 
 To install and register your host using Puppet, or manually, see https://access.redhat.com/products/red-hat-insights/#getstarted[Red{nbsp}Hat Insights Getting Started].
 
+.{Team} Insights Information Available for Hosts
+Additional information is available about hosts through {Team} Insights.
+
+You can find this information in two places.
+
+* In the {ProjectWebUI}, navigate to *Configure > Insights* where the three dots icon next to the *Remediate* button provides a *View in {Team} Insights* link to the general recommendations page.
```

Comment:
Do we have a better term for the "three dots icon"? If it's the "kebab" button it does have a term, I forgot it again, sorry... please check the IBM style guide.

Reply:
Checking with UX team

Reply:
I would go with "vertical ellipsis".
Screenshot from the [PF4 page](https://www.patternfly.org/v4/guidelines/icons/#font-awesome-solid-fas-vs-font-awesome-regular-far). Also, in the PF4 design system is normal to use the term "[kebab](https://www.patternfly.org/v4/components/dropdown/html/#kebab)" - which is probably not the best solution for documentation purposes.
<img width="888" alt="Screenshot 2022-04-28 at 11 29 33" src="https://user-images.githubusercontent.com/51960547/165723940-6195af80-d510-4901-9265-6db6540ee537.png">